pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -657,7 +857,7 @@ func (r *ReconcileLogStorage) Reconcile(ctx context.Context, request reconcile.R
 			return result, err
 		}
 
-		result, proceed, err = r.createEsMetrics(
+		result, proceed, err = r.createESMetrics(
```

Comment:
clusterdomain is never used anymore.

Reply:
it is - it's passed to the render code and used there